tests/network/primary_pod_network.go

```diff
@@ -52,7 +52,10 @@ var _ = SIGDescribe("Primary Pod Network", func() {
 			By("Making sure IP reported on the VMI matches the one on the pod")
 			Expect(vmi.Status.Interfaces[0].IP).To(Equal(vmiPod.Status.PodIP))
 			Expect(vmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
-			Expect(vmi.Status.Interfaces[0].IPs[0]).To(Equal(vmiPod.Status.PodIP))
+			Expect(len(vmi.Status.Interfaces[0].IPs)).To(Equal(len(vmiPod.Status.PodIPs)))
+			for i, ip := range vmiPod.Status.PodIPs {
+				Expect(vmi.Status.Interfaces[0].IPs[i]).To(Equal(ip.IP))
```

Comment:
One thing I ended up thinking: is it possible that the IPs end up in a different order ?

I think we only have one IP of each family, and IPv4 will appear first, but ... Is that possible ? 

Even if not, would it make sense to 'prepare' in advance for it ? (I guess it depends **if** we want to ever have multiple IPs of the same family. I don't know if we do).

Reply:
the Ips cant end in different order, because:

1. K8s API enforce that PodIPs[0] must be equal to PodIP
Even if ipv6 would be able to be the primary, still this will be the order of both arrays.
and the core code of 3958 will take care of it already.

K8s doesnt allow multi IPs per family for the primary interface, 3958 handles it.

Reply:
Thanks, just wanted to be sure.